integration-tests/cstor-pool/cstor_integration_test.go

```diff
@@ -335,6 +335,100 @@ var _ = Describe("Integration Test", func() {
 			}
 			Expect(onlineCspCount).To(Equal(3))
 
+		})
+	})*/
```

Comment:
Would you please check what */ is?
I think this is not required.

Reply:
Yeah, this might have come up while rebasing